packages/uiweb/src/lib/components/chat/ChatViewBubble/ChatViewBubble.tsx

```diff
@@ -173,94 +186,188 @@ const MessageWrapper = ({
 
 export const ChatViewBubble = ({
   decryptedMessagePayload,
-  isGroup,
+  chatPayload,
+  chatReactions,
+  showChatMeta,
+  chatId,
+  actionId,
+  singularActionId,
+  setSingularActionId,
 }: {
   decryptedMessagePayload: IMessagePayload;
-  isGroup: boolean;
+  chatPayload?: IMessagePayload;
+  chatReactions?: any;
+  showChatMeta?: boolean;
+  chatId?: string;
+  actionId?: string | null | undefined;
+  singularActionId?: string | null | undefined;
+  setSingularActionId?: (singularActionId: string | null | undefined) => void;
 }) => {
+  // get theme
+  const theme = useContext(ThemeContext);
+
+  // TODO: Remove decryptedMessagePayload in v2 component
+  if (!chatPayload) {
+    chatPayload = decryptedMessagePayload;
+  }
+
+  if (!showChatMeta) {
+    showChatMeta = false;
+  }
```

Comment:
Agreed though this might break backward compatibility since the only required parameter for v1 for ChatViewBubble is decryptedMessagePayload: https://push.org/docs/chat/ui-components/Advanced/chatviewbubble-component/

Any suggestions?

Reply:
@HarshRajat Added couple of comments.